packages/fetchai/connections/p2p_libp2p/aea/envelope.proto

```diff
@@ -1,5 +1,6 @@
 syntax = "proto3";
 
+//package libp2p_node.aea;
 package aea;
```

Comment:
need to find a way to import this proto file from `message.proto` one to avoid double serialization of `aea.Envelope`